docs/api.rst

```diff
@@ -4917,8 +4917,9 @@ Dealing with History Events
                       "timestamp": 1642802807
                   },
                   "customized": false,
+                  "ignored_in_accounting": false,
                   "has_details": false,
-		  "grouped_events_num": 1
+                  "grouped_events_num": 1
```

Comment:
this change is because the line was using tab instead of spaces for indentation

Reply:
Thanks. I think most of these mistakes is due to some fucked up settings in my editor for rst